runtime/doc/options.txt

```diff
@@ -1030,7 +1030,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option lets you choose which characters might cause a line
 	break if 'linebreak' is on.  Only works for ASCII characters.
 
-						*'breakindent'* *'bri'*
+				*'breakindent'* *'bri'* *'nobreakindent'* *'nobri'*
```

Comment:
A Tab character was removed in <https://github.com/vim/vim/commit/7c1c6dbb6817640fd3956a0d5417da23fde336d8>
```suggestion
			*'breakindent'* *'bri'* *'nobreakindent'* *'nobri'*
```